src/sass/components/_about.scss

```diff
@@ -0,0 +1,93 @@
+.about {
+  background-color: $pageBg;
+  background: url('../images/sectionbg1_1x.png') no-repeat;
+  background-size: contain;
+  background-position: center bottom;
+
+  @media (min-width: 768px) {
+    background-image: none;
+  }
+  @media (min-width: 1200px) {
+    background-image: url('../images/sectionbg1_3x.png');
+    background-position-y: 140%;
+  }
+
+  &__tittle-2th {
+    @include font-mixin(Titan One, null, 34, 39px, null, $brand-red);
+    text-align: Center;
+    vertical-align: Top;
+    padding-bottom: 26px;
+
+    @media (min-width: 768px) {
+      font-size: 48;
+    }
+    @media (min-width: 1200px) {
+      font-size: 58;
+    }
+  }
+
+  &__title {
+    @include font-mixin(Titan One, null, 18, 21px, null, $brand-pink);
+    display: block;
+    text-align: center;
+    padding-top: 125px;
+    padding-bottom: 15px;
+  }
+
+  &__img {
+    width: 280px;
+    height: 279px;
+    padding-bottom: 46px;
+    object-fit: contain;
+
+    @media (min-width: 768px) {
+      margin-right: 17px;
+      width: 340px;
+      height: 339px;
+    }
+    @media (min-width: 1200px) {
+      margin-right: 74px;
+      width: 516px;
+      height: 516px;
+    }
+  }
+
+  &__wrapper {
+    display: flex;
+    align-items: center;
+    flex-direction: column;
+
+    @media (min-width: 768px) {
+      flex-direction: row;
+      justify-content: center;
+    }
+  }
+
+  &__text-wrapper {
+    @media (min-width: 768px) {
+      max-width: 340px;
+    }
+  }
+
+  &__text {
+    @include font-mixin(DM Sans, null, 12, 19px, null, null);
```

Comment:
font-family musisz dać w pojedynczy cudzysłów: 'test'
font-weight ma 500, a nie null
font-size popraw na 12px
line height obliczasz dzieląc line height/font-size, czyli 19/12, to wychodzi ~1.58 i tak zapisz
letter- spacing zrób normal
color zrób $black

Reply:
color zostawiam narazie na text bo nie mam zaciagnietej nowej zmiennej black i kompilator wyrzuca mi blad